rust/agama-server/src/questions.rs

```diff
@@ -94,7 +95,7 @@ trait AnswerStrategy {
     ///
     /// I gets as argument the question to answer. Returned value is pair
     /// of `answer` and `password` properties. If `None` is used in any
-    /// position it means that this object does not respond to given property.
```

Comment:
np: typo

Reply:
thanks for catch. I have to say that sometime vscode together with middle button of mouse do serious mess.